src/bin/zcurve.rs

```diff
@@ -20,64 +20,64 @@ fn zcurve(img: &mut Image, x_offset: i64, y_offset: i64) {
         if next & 0b100 > 0 {
             sx += 2 * STEP_SIZE;
         }
-        if next & 0b10_000 > 0 {
```

Comment:
This one is for readability, since the bit pattern in the zcurve is 3 bits long. It wasn't accidental. I prefer leaving this in, even if the linter complains

Reply:
reverted and allowed